awssdk/src/main/scala/implicits.scala

```diff
@@ -12,9 +12,10 @@ import cats.implicits._
 import meteor.codec.{Decoder, DecoderFailure}
 import software.amazon.awssdk.services.dynamodb.model.AttributeValue
 
-object implicits {
+object implicits extends syntax {
   type FailureOr[U] = Either[DecoderFailure, U]
 
+  // TODO I would keep it private
```

Comment:
agree, let's make this whole object `private` to the package

Reply:
Uhm the syntax is needed to define encoders and decoders. 

I was thinking that was good for the final user to be able to do 

```import meteor.implicits._```

and being able to use the syntax on the AttributeValue